### PR TITLE
[dv/rstmgr] Add alert and cpu_info_attr assertions

### PR DESCRIPTION
**Pull request by matutem**
Also make minor fixes to docs.
Remove common CSR test exclusion from sw_rst_regwen.

Signed-off-by: Guillermo Maturana <maturana@google.com>

### CODE REVIEW

File: hw/top_earlgrey/ip/rstmgr/data/autogen/rstmgr.hjson [hunk 1]
```diff
@@ -356,10 +356,8 @@
             resval: "1",
           },
         ],
-        tags: [// Don't reset other IPs as it will affect CSR access on these IPs
-               "excl:CsrAllTests:CsrExclWrite"]
       }
-    }
+    },
```
Comment (weicaiyang) [inline hw/top_earlgrey/ip/rstmgr/data/autogen/rstmgr.hjson:360]: this comma seems not needed.

**Review by weicaiyang — COMMENTED** [review]
we have design change on the reg name. @tjaychen should take a look too.

File: hw/ip/rstmgr/data/rstmgr.hjson [hunk 2]
```diff
@@ -219,7 +219,7 @@
 
     { multireg: {
         cname: "RSTMGR_SW_RST",
-        name:  "SW_RST_REGEN",
+        name:  "SW_RST_REGWEN",
```
Comment (tjaychen) [inline hw/ip/rstmgr/data/rstmgr.hjson:222]: that's weird..i thought i already merged a PR to do this...

Comment (matutem) [reply hw/ip/rstmgr/data/rstmgr.hjson:222]: Apparently you just did the .tpl / autogen files.

Comment (matutem) [reply hw/ip/rstmgr/data/rstmgr.hjson:222]: And this only makes the names consistent with the top_earlgrey names, so no real name change.

File: hw/top_earlgrey/ip/rstmgr/data/autogen/rstmgr.hjson (hunk 1 above)
Comment (matutem) [reply hw/top_earlgrey/ip/rstmgr/data/autogen/rstmgr.hjson]: Done

**Review by weicaiyang — COMMENTED** [review]
DV part change LGTM, defer to @tjaychen for approval, as I'm not sure if the design change is expected.

File: hw/ip/rstmgr/data/rstmgr.hjson (hunk 2 above)
Comment (tjaychen) [reply hw/ip/rstmgr/data/rstmgr.hjson:222]: o sounds good.

**Comment by tjaychen** [comment]
LGTM

**Review by weicaiyang — APPROVED** [review]

**Review by tjaychen — APPROVED** [review]

**Pull request merged**